spec/live/custom_status_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'core/spec_helper'
+require 'securerandom'
+
+describe ZendeskAPI::CustomStatus, :delete_after do
+  def valid_attributes
+    {
+      :status_category => 'open',
+      :agent_label => "Agent Label #{SecureRandom.hex(6)}",
+      :end_user_label => "End User Label #{SecureRandom.hex(6)}",
+      :description => "Description #{SecureRandom.hex(6)}",
+      :end_user_description => "End User Description #{SecureRandom.hex(6)}",
+      :active => false
+    }
```

Comment:
Minor: Ideally, we would use the more elegant Ruby 1.9 hash syntax:

```suggestion
    {
      status_category: 'open',
      agent_label: "Agent Label #{SecureRandom.hex(6)}",
      end_user_label: "End User Label #{SecureRandom.hex(6)}",
      description: "Description #{SecureRandom.hex(6)}",
      end_user_description: "End User Description #{SecureRandom.hex(6)}",
      active: false
    }
```

Reply:
@ecoologic Syntax updated in https://github.com/zendesk/zendesk_api_client_rb/pull/491/commits/df8a7f4295dbe5ea635970d262ca16b8bb7edeb4